### DISCUSSION TITLE
How do I enable multiple dependencies for multiple platforms (x86 and x64) inside of a manifest file?

### DISCUSSION THREAD

**Discussion by ughstudios** [microsoft/vcpkg#25033] [Q&A]
```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
  "name": "OverlayInjector",
  "version-string": "0.0.1",
  "dependencies": [
    "boost-program-options",
    "boost-program-options:x86-windows-static",
    "spdlog",
    "spdlog:x86-windows-static"
  ]
}
```

I have the above manifest file but it does not seem to work. I want spdlog 64 and x86 builds. I'm using a Visual Studio CMake project so I would like it to download the x64 version when my build configuration is set to x64-Debug/Release and x86 when it's set to x86-Debug/Release

What could I do differently?

**Comment by autoantwort** [comment]
You don't have to. Simply use the cmake integration by setting the toolchain file.

**Reply by AugP** [reply]
> You don't have to. Simply use the cmake integration by setting the toolchain file.

This. Visual Studio CMake projects have built-in support for vcpkg. As long as you ran "vcpkg integrate install" and your CMake is referencing the vcpkg.cmake file it should work. When you hit build vcpkg will use your project's active build configuration to figure out which triplets to use for your dependencies. 

See https://vcpkg.io/en/docs/users/buildsystems/cmake-integration.html for more information as well.
